### PR TITLE
Clarification for InlineValue request

### PR DESCRIPTION
**Pull request by haukot**
Based on the current description of the InlineValue request, it gives the impression that the LSP server should compute the values, and the editor simply shows them. But it actually returns not the values, but the information (variable names, expressions) based on which the editor should compute the values.

I got confused by this at first, and have tried to rephrase it to be more aligned with the actual feature.

References that helped me to understand:
* https://github.com/microsoft/language-server-protocol/pull/1318#issuecomment-895292256
* https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_54.md#inline-value-provider-api
* https://github.com/microsoft/vscode/issues/105690

### CODE REVIEW

**Comment by haukot** [comment]
@microsoft-github-policy-service agree

**Comment by haukot** [comment]
I stumbled upon this again today. Anything I can do to help push it forward?

**Review by dbaeumer — APPROVED** [review]

**Comment by dbaeumer** [comment]
Thanks. The clarifications make perfect sense.

**Review by rzhao271 — APPROVED** [review]
